client/coral-embed-api/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-var */
+/**
+ * @file Exposes the main 'window.Coral.Talk' API that developers can use to
```

Comment:
Let's call this `coral-embed` as this is the common entry point for `coral-embed-*`.